api/src/main/java/org/pmiops/workbench/impersonation/ImpersonatedWorkspaceServiceImpl.java

```diff
@@ -163,6 +167,7 @@ public void deleteWorkspace(
 
     workspaceDao.saveWithLastModified(
         dbWorkspace.setWorkspaceActiveStatusEnum(WorkspaceActiveStatus.DELETED), dbUser);
+    featuredWorkspaceDao.deleteDbFeaturedWorkspaceByWorkspace(dbWorkspace);
```

Comment:
we have separate entry for featured workspace in addition to an entry in the workspace table, and this is to clean up the featured workspace entry if it exists?

Reply:
That's right.  We don't benefit from a CASCADE because we are not actually deleting the `dbWorkspace` entry.  See the previous PR #8733 for a similar example.